Vagrantfile

```diff
@@ -34,63 +34,67 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+        :salt_repo => "salt-repo-2017.7-1.el.repo",
+        :optional_states => "oracle-java",
+        :custom_image_type => "hortonworks",
+        :oracle_jdk8_url_rpm => "http://download.oracle.com/otn-pub/java/jdk/8u151-b12/e758a0de34e24606bca991d704f6dcbf/jdk-8u151-linux-x64.rpm"
       },
-      # {
-      #   :hostname => "centos6-vagrant",
-      #   :box => "centos/6",
-      #   :ram => 1536,
-      #   :cpu => 2,
-      #   :ambari_baseurl => "http://s3.amazonaws.com/dev.hortonworks.com/ambari/centos6/2.x/BUILDS/2.4.1.0-22/",
-      #   :ambari_key => "http://s3.amazonaws.com/dev.hortonworks.com/ambari/centos6/RPM-GPG-KEY/RPM-GPG-KEY-Jenkins",
-      #   :hdp_stack_version => "2.5",
-      #   :hdp_version => "2.5.0.1-60",
-      #   :hdp_baseurl => "http://s3.amazonaws.com/dev.hortonworks.com/HDP/centos6/2.x/BUILDS/2.5.0.1-60",
-      #   :hdp_repoid => "HDP-2.5",
-      #   :salt_repo => "https://repo.saltstack.com/yum/redhat/salt-repo-latest-2.el6.noarch.rpm"
-      # },
-      # {
-      #   :hostname => "wheezy-vagrant",
-      #   :box => "debian/wheezy64",
-      #   :ram => 1536,
-      #   :cpu => 2,
-      #   :ambari_baseurl => "http://s3.amazonaws.com/dev.hortonworks.com/ambari/debian7/2.x/BUILDS/2.4.1.0-22/",
-      #   :ambari_key => "B9733A7A07513CAD",
-      #   :salt_repo => "http://repo.saltstack.com/apt/debian/7/amd64/latest"
-      # },
-      # {
-      #   :hostname => "xenial-vagrant",
-      #   :box => "ubuntu/xenial64",
-      #   :ram => 1536,
-      #   :cpu => 2,
-      #   :ambari_baseurl => "http://s3.amazonaws.com/dev.hortonworks.com/ambari/ubuntu12/2.x/BUILDS/2.4.1.0-22/",
-      #   :ambari_key => "B9733A7A07513CAD",
-      #   :salt_repo => "http://repo.saltstack.com/apt/ubuntu/12.04/amd64/latest"
-      # },
-      # {
-      #   :hostname => "trusty-vagrant",
-      #   :box => "ubuntu/trusty64",
-      #   :ram => 1536,
-      #   :cpu => 2,
-      #   :ambari_baseurl => "http://s3.amazonaws.com/dev.hortonworks.com/ambari/ubuntu14/2.x/BUILDS/2.4.1.0-22/",
-      #   :ambari_key => "B9733A7A07513CAD",
-      #   :salt_repo => "http://repo.saltstack.com/apt/ubuntu/14.04/amd64/latest"
-      # },
+      {
+        :hostname => "centos6-vagrant",
+        :box => "centos/6",
+        :ram => 1536,
+        :cpu => 2,
+        :salt_repo => "salt-repo-2016.11-6.el.repo",
+        :custom_image_type => "hortonworks",
+        :oracle_jdk8_url_rpm => "http://download.oracle.com/otn-pub/java/jdk/8u151-b12/e758a0de34e24606bca991d704f6dcbf/jdk-8u151-linux-x64.rpm"
+      },
+      {
+        :hostname => "wheezy-vagrant",
+        :box => "debian/wheezy64",
+        :ram => 1536,
+        :cpu => 2,
+        :salt_repo => "salt-repo-2016.11-5.debian7.list",
+        :custom_image_type => "hortonworks"
+      },
+      {
+        :hostname => "precise-vagrant",
+        :box => "ubuntu/precise64",
+        :ram => 1536,
+        :cpu => 2,
+        :salt_repo => "salt-repo-2016.11-3.ubuntu12.list",
```

Comment:
as above

Reply:
No newer repository is available for Ubuntu 12.04.
http://repo.saltstack.com/apt/ubuntu/12.04/amd64/archive/

Reply:
Ubuntu 12.04 is not supported by Canonical: https://wiki.ubuntu.com/Releases

Reply:
You are right but HDP is still supported on Ubuntu 12.04. I have the same problem with Debian 7 (support ends on 31st May 2018).